### ISSUE TITLE
Epitath implant

### ISSUE THREAD

**Issue by rossark** [citadel-station/citadel-main#14090]

**Comment by MalricB** [comment]
yep, I fucked up and gave only the action button the appearance switch. latest version should have the appearance changes into the normal switch + work on the action button too

**Comment by ghost** [comment]
@MalricB did you open a new pr for it, any changes after your pr merged require an entirely new pr and i didn't spot one but that might just be my bad

**Comment by MalricB** [comment]
> @MalricB did you open a new pr for it, any changes after your pr merged require an entirely new pr and i didn't spot one but that might just be my bad

it's still testmerged isn't it? or am I blind and retarded

**Comment by ghost** [comment]
no i'm blind and retarded i just forgot people don't merge features here
